requirements/dev-extra.in

```diff
@@ -8,6 +8,6 @@
 -c constraints.txt
 
 pluggy==0.8.1
-tox==3.7.0
-tox-battery==0.5.2
+tox
+tox-battery
```

Comment:
This is what started adding the unpinned dependencies to the output files, but we shouldn't need to keep updating this by hand.